src/server/migrations/20210118154457_create_user_table.js

```diff
@@ -0,0 +1,19 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('user', function(table) {
```

Comment:
Doesn't it have to be `users`, with an -s? As in users.controller.js  we have fx.: `return await knex('users').select('users.id', 'users.title');` 
So I think there will be an error if we give the table the name of**user**, but in the other files we make a call to the table **users**

Reply:
my table also doesnt have title and there are other collumns.. i did the controller file before i saw the diagram for DB. now its a whole mess.. i think i can keeo user table and then someone coud work on the controller story again?

Reply:
that story was closed and PR merged before i made the seed/migrations story where I realized there was a diagram for DB. So maybe you would wanna re-open the controller story and add all the collumns in controller that are also in the diagram? And change the table name to user there? [diagram here](https://dbdiagram.io/d/5ffc249780d742080a35cb4a)